### ISSUE TITLE
latex not rendering in docs 

### ISSUE THREAD

**Issue by peekxc** [DavidLapous/multipers#18]
The latex is not rendering in the documentation for me 

<img width="739" alt="Screenshot 2024-09-15 at 9 26 13 AM" src="https://github.com/user-attachments/assets/de8621a6-7903-4b0c-9c94-580451f5eb30">

Inspecting the source, it seems the latex source is just rendering as text, i.e. it's not a MathML / MathJax / KaTeX related issue

<img width="224" alt="Screenshot 2024-09-15 at 9 26 45 AM" src="https://github.com/user-attachments/assets/f3e820c3-a2b8-48fb-86d3-5b9338dae92a">

Related [#6773](https://github.com/openjournals/joss-reviews/issues/6773)

**Comment by peekxc** [comment]
Interestingly, this seems to only affect some of the docs, as the docs on the main page render fine: 

<img width="701" alt="Screenshot 2024-09-15 at 9 31 26 AM" src="https://github.com/user-attachments/assets/317c56e6-c632-4849-92fc-8c7698df68f8">

**Comment by DavidLapous** [comment]
Dear @peekxc , thanks for your investigation!
I think it is because I activated a "lazy" update of the documentation which doesn't recompile ---and simply reads--- 
the notebooks in the documentation, which seems to prevent sphinx from compiling latex? 
IIRC, I did that to have a faster compilation of the library in my scripts (which is already quite long),
this should be fixed by removing the lines in and recompiling the doc
https://github.com/DavidLapous/multipers/blob/3f664fff99cc7576019a698eda76f9e0efb4885f/docs/conf.py#L34-L36

I'll do that for the next release.
